src/frontend/utils/behandling.ts

```diff
@@ -30,3 +30,14 @@ export const hentSøkersMålform = (behandling: IBehandling) =>
     })?.målform ?? Målform.NB;
 
 export const MIDLERTIDIG_BEHANDLENDE_ENHET_ID = '4863';
+
+export const erBehandlingMedVedtaksbrevutsending = (åpenBehandling: IBehandling) => {
+    const erMigreringFraInfotrygd = åpenBehandling.type === Behandlingstype.MIGRERING_FRA_INFOTRYGD;
+
+    return (
+        åpenBehandling.type !== Behandlingstype.TEKNISK_ENDRING &&
+        åpenBehandling.årsak !== BehandlingÅrsak.SATSENDRING &&
+        åpenBehandling.årsak !== BehandlingÅrsak.SMÅBARNSTILLEGG_ENDRING_FRAM_I_TID &&
+        !erMigreringFraInfotrygd
```

Comment:
Hva med noe ala dette i stedet? `![Behandlingstype.TEKNISK_ENDRING, Behandlingstype. MIGRERING_FRA_INFOTRYGD].includes(åpenBehandling.type) && ![BehandlingÅrsak.SATSENDRING, BehandlingÅrsak.SMÅBARNSTILLEGG_ENDRING_FRAM_I_TID].includes(åpenBehandling.årsak)`

Reply:
ikke sikkert det er mer ryddig da, men uansett kanskje sjekke migrering fra infotrygd direkte som de andre årsakene/typene

Reply:
Endret i [Flytt alert ut av Vedtaksbrev. Flytt Vedtaksmeny under if om erVedtak…](https://github.com/navikt/familie-ba-sak-frontend/pull/3053/commits/0b4192a79fd7d8191f23c1d55622c27baa34e716)

Reply:
Og denne for å få det riktig 😅 
[Bruk riktig type for brev som ikke skal sendes ut](https://github.com/navikt/familie-ba-sak-frontend/pull/3053/commits/21f9b449d4ef0d9fe72fcb0e1537b6d685d6f517)